tests/main.rs

```diff
@@ -944,12 +944,12 @@ async fn test_arrow_nested_double_columns() {
     assert_batches_eq!(expected, &batches);
 }
 
-#[tokio::test]
-async fn test_lexical_precedence_wrong() {
-    let sql = r#"select '{"a": "b"}'->>'a'='b' as v"#;
-    let err = run_query(sql).await.unwrap_err();
-    assert_eq!(err.to_string(), "Error during planning: Unexpected argument type to 'json_as_text' at position 2, expected string or int, got Boolean.")
-}
+// #[tokio::test]
+// async fn test_lexical_precedence_wrong() {
+//     let sql = r#"select '{"a": "b"}'->>'a'='b' as v"#;
+//     let err = run_query(sql).await.unwrap_err();
+//     assert_eq!(err.to_string(), "Error during planning: Unexpected argument type to 'json_as_text' at position 2, expected string or int, got Boolean.")
+// }
```

Comment:
@samuelcolvin I assume this test was documenting broken parse precedence and we can now just remove it?

Reply:
(Or update it to test the successful case?)

Reply:
yes, I'll fix.